assets/src/sass/3-utilities/_animations.scss

```diff
@@ -10,7 +10,8 @@
 	animation-timing-function: ease-out;
 }
 
-@keyframes fadeInLeft {
+@keyframes fade-in-left {
```

Comment:
Not sure if we used this animation, better to keep only a single example like `fade-in` and remove other all styles. Whenever project-specific animation required, the dev can add it to this file.